module/solutions/helper.js

```diff
@@ -172,6 +172,10 @@ module.exports = class SolutionsHelper {
             solutionData.entities = entitiesData;
           }
 
+          if( !solutionData.minNoOfSubmissionsRequired || !solutionData.allowMultipleAssessemts ) {
```

Comment:
@priyanka-TL change the condition to if given in request and greater than 1 set else do nothing.